go-controller/pkg/util/nicstobridge.go

```diff
@@ -218,11 +219,16 @@ func NicToBridge(iface string) (string, error) {
 	setupDefaultFile()
 
 	// Get ip addresses and routes before any real operations.
-	addrs, err := netLinkOps.AddrList(ifaceLink, syscall.AF_INET)
+	family := syscall.AF_UNSPEC
+	// FIXME: Should move all interfaces over, but in IPv4 only, breaks.
```

Comment:
how does it break?

oh, you probably only want to move the `.IsGlobalUnicast()` IPs over... (ie, in particular you don't want to try to move IPv6 link-local addresses)

Reply:
(This could happen later though if you want to get the current version in now)

Reply:
Added the `.isGlobalUnicast()`, but still need this hack.

Reply:
dont worry too much about this. I think only KIND actually moves the interface. OpenShift doesn't and neither does Nvidia. So whatever hack you are doing here is OK for now.